cmd/thor/utils.go

```diff
@@ -712,3 +711,14 @@ func parseNodeList(list string) ([]*discover.Node, error) {
 
 	return nodes, nil
 }
+
+func readIntFromFlag(ctx *cli.Context, name string) (int, error) {
```

Comment:
I think it's a good idea to normalize this, perhaps the behaviour can be simplified further by passing the uint64 ?

```suggestion
func readIntFromUInt64Flag(val uint64) (int, error) {
```

Reply:
The name of the flag was used for a better error hint for that, that's why it comes with the context, naming seems better, but do you consider passing `(val uint64, name string)` is better?